casbin/persist/adapters/file_adapter.py

```diff
@@ -59,4 +59,10 @@ def remove_policy(self, sec, ptype, rule):
         pass
 
     def remove_policies(self,sec,ptype,rules):
+        pass
+
+    def update_policy(self, sec, ptype, old_rule, new_rule):
```

Comment:
These interfaces exist in BatchAdapter, no need for this.

Reply:
Ok, I konw what you mean. I first think it's used for normal file adapter, because I test it in the test_management_api.py and it throw a wrong message.

Reply:
Can I delete these two file changes (file_adapter.py and test_management.py) directly?

Reply:
Just push to `sheny1xuan:model_update`, all the change will be synced to Pull Request.